test/library/input_numeric_format.spec.js

```diff
@@ -48,6 +48,17 @@ describe('Test NumberFormat as input with numeric format options', () => {
     expect(wrapper.state().value).toEqual('-$123.55');
   });
 
+  it('supports negative numbers when using parentheses negationFormat', () => {
+    const wrapper = shallow(<NumberFormat thousandSeparator={true} prefix={'$'} negationFormat={'parentheses'} />);
+
+    simulateKeyInput(wrapper.find('input'), '-', 0);
+
+    expect(wrapper.state().value).toEqual('-');
```

Comment:
Is this expected that on typing `-` first, it will be `-` and after typing number, it will change to parenthesis. I will see what is the expected behavior on a google sheet or MS Excel.

Reply:
I feel whenever we press -, it should convert it to `()` even though there are no numbers yet

Reply:
Good point, I like it that way better too.
Done!